mathjax2/legacy/extensions/TeX/AMSmath.js

```diff
@@ -24,9 +24,9 @@
  *  limitations under the License.
  */
 
-let sm = require('mathjax3/input/tex/SymbolMap.js');
-let tc = require('mathjax3/input/tex/TexConstants.js');
-let BaseMethods = require('mathjax3/input/tex/BaseMethods.js').default;
+let sm = require('../../../../mathjax3/input/tex/SymbolMap.js');
+let tc = require('../../../../mathjax3/input/tex/TexConstants.js');
+let BaseMethods = require('../../../../mathjax3/input/tex/BaseMethods.js').default;
 
```

Comment:
Hi @dpvc!

Can you explain the issue you had with the paths above a bit more?

Reply:
@jdalton,

Because v3 might be included into an application via `npm` (node package manager), the top-level directory might be `node_modules/mathjax-v3/mathjax3` or some such thing.  Using relative directories makes MathJax v3 able to be used from within whatever folder hierarchy the author is using, rather than expecting it to be in a particular location as the original form does.

Reply:
@jdalton, with `node -r esm samples/tex2html.js` perhaps the root directory is considered to be `./samples` since that is where `tex2html.js` is being loaded?  So since `mathjax3` is in the main directory (a sibling of `samples`) and not in `samples` itself, it isn't being found?

Reply:
I think this may be an issue on our end. Are you using symlinks?